src/components/templates/authentication/register.jsx

```diff
@@ -59,16 +59,16 @@ const Register = () => {
             <div className="relative mt-4 flex items-center">
               <div className="flex h-6 items-center">
                 <input
-                  id="notifications"
-                  aria-describedby="notifications-description"
-                  name="notifications"
+                  id="agree-to-receive-email"
```

Comment:
No need to add such long ids, we can go with a common convention. For example, `checkbox-1`, `checkbox-2`and so on. Same for `name` and `area-describedby`

Reply:
I have changed the ID and names

![image](https://github.com/hiteshchoudhary/devui/assets/99522246/ba8e1c80-e483-431b-ade8-887c22479c8f)